packages/webapp/src/components/PopupFilter/PureCollapsingSearch/index.jsx

```diff
@@ -22,13 +22,14 @@ import Input from '../../Form/Input';
 import TextButton from '../../Form/Button/TextButton';
 import { Modal } from '../../Modals';
 
-export default function PureCollapsibleSearch({
+export default function PureCollapsingSearch({
   value,
   isSearchActive,
   onChange,
   placeholderText,
   className,
   containerRef,
+  isDesktop,
```

Comment:
It was a media query -- no we should not use media query I don't think. 

- `mixin.scss` has breakpoints that are OK since we can change the whole app at the same time. If this is the preferred choice I can change 🤷 .

- isDesktop is nice in my personal opinion because:
  - we copied MUI breakpoints to make the `mixin.scss` so now we have 2 copies of source of truth
  - Using good CSS class naming like [BEM](https://medium.com/trabe/a-comprehensive-guide-to-using-bem-with-react-14d00c199e0d) I think javascript files can be slightly more readable versus hidden in CSS (I thought this was blocker so I rushed naming but fixed to BEM) ![Screenshot 2024-03-12 at 4 30 27 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/856bdb7c-4964-4a59-9409-d0cf61550f2f)
  -  sometimes we want to use isDesktop for in page content or design decisions too so I enjoy letting MUI control our design system as a whole (since they are the experts)
![Screenshot 2024-03-12 at 4 31 20 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/9df1462a-d931-4221-b7a3-c400bf8cd0b6)

Reply:
I really like having the breakpoints being controlled by JS/TS and no longer the mixins... those multiple methods and sources of truth (when finances was created we weren't yet unified around the MUI breakpoints) made it really hard to reason about e.g. the CollapsibleSearch.

But GOSH I hate those BEM underscores... do we have to?? 🤣

Reply:
lol no not necessary -- but its a good architecture vs "willy nilly" haha

Reply:
Okay, I was in favour of using CSS. I don't think we have discussed which strategy to stick with? I added to the tech daily agenda!

Reply:
@SayakaOno sounds awesome! I don't have a strong opinion between mixins and inline breakpoints just no more media queries haha

Reply:
I don't think we made a decision today. @SayakaOno can I make a tech debt ticket to choose ? That way search bar can go through.

Reply:
I thought we wouldn't need any further discussion 😅 It's up to you!

Reply:
I think it would be nice to choose a direction haha: https://lite-farm.atlassian.net/browse/LF-4143